src/connectionbenchmark/benchmarktaskdownload.cpp

```diff
@@ -55,43 +55,60 @@ void BenchmarkTaskDownload::handleState(BenchmarkTask::State state) {
   }
 }
 
+void BenchmarkTaskDownload::connectNetworkRequest(NetworkRequest* request) {
```

Comment:
If we know that QDnsLoop does not work on android, can we simply add a:
```
#ifdef MVPN_ANDROID
  NetworkRequest* request =
      NetworkRequest::createForGetUrl(this, m_fileUrl.toString());
  connectNetworkRequest(request);
#endif
....

Reply:
Removed the fallback.